### ISSUE TITLE
SmartCity-Accessibility #344 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#200]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 192 lines...]
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078436 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 12.576 sec - in smartcity.accessibility.navigation.NavigationTest

Results :

Failed tests: 
  ReviewManagerTest.updateReviewTest:175 null

Tests in error: 
  ReviewManagerTest.getReviewByUserAndLocationTest:81 » IndexOutOfBounds Index: ...

Tests run: 52, Failures: 1, Errors: 1, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:59 min
[INFO] Finished at: 2017-01-22T12:50:16+00:00
[INFO] Final Memory: 28M/180M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [assaflu] c17e82e3e846c7a95ad5a293255fff405492bcd8 - fixed few bugs

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/344/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
